components/coda/actions/list-docs/list-docs.mjs

```diff
@@ -0,0 +1,121 @@
+import coda from "../../coda.app.mjs";
+
+export default {
+  key: "coda_list-docs",
+  name: "List Docs",
+  description: "Returns a list of Coda docs accessible by the user. These are returned in the same order as on the docs page: reverse chronological by the latest event relevant to the user (last viewed, edited, or shared).",
+  version: "0.0.26",
+  type: "action",
+  props: {
+    coda,
+    isOwner: {
+      propDefinition: [
+        coda,
+        "isOwner",
+      ],
+    },
+    isPublished: {
+      propDefinition: [
+        coda,
+        "isPublished",
+      ],
+    },
+    query: {
+      propDefinition: [
+        coda,
+        "query",
+      ],
+    },
+    sourceDoc: {
+      propDefinition: [
+        coda,
+        "sourceDoc",
+        (c) => c,
+      ],
+      description: "Show only docs copied from the specified doc ID.",
+    },
+    isStarred: {
+      propDefinition: [
+        coda,
+        "isStarred",
+      ],
+    },
+    inGallery: {
+      propDefinition: [
+        coda,
+        "inGallery",
+      ],
+    },
+    workspaceId: {
+      propDefinition: [
+        coda,
+        "workspaceId",
+      ],
+    },
+    folderId: {
+      propDefinition: [
+        coda,
+        "folderId",
+      ],
+      description: "Show only docs belonging to the given folder.",
+    },
+    limit: {
+      propDefinition: [
+        coda,
+        "limit",
+      ],
+    },
+    pageToken: {
+      propDefinition: [
+        coda,
+        "pageToken",
+      ],
+    },
+    paginate: {
+      propDefinition: [
+        coda,
+        "paginate",
+      ],
+    },
+  },
+  async run() {
+    var params = {
+      isOwner: this.isOwner,
+      isPublished: this.isPublished,
+      query: this.query,
+      sourceDoc: this.sourceDoc,
+      isStarred: this.isStarred,
+      inGallery: this.inGallery,
+      workspaceId: this.workspaceId,
+      folderId: this.folderId,
+      limit: this.limit,
+      pageToken: this.pageToken,
+    };
+    Object.keys(params).forEach((key) => (params[key] === null
+      || params[key] === undefined
+      || params[key] === "")
+      && delete params[key]);
+
+    var result = await this.coda.listDocs(params);
+
+    if (!this.paginate) {
```

Comment:
For now you can remove the `paginate` prop and always paginate / return all items to the user. This is the common use case, and we're trying to remove the complexity of needing to paginate. We may revisit this in the future and expose the pagination / offset params directly to the user.

Reply:
Issue addressed